gdcdictionary/schemas/core_metadata_collection.yaml

```diff
@@ -373,6 +373,11 @@ properties:
       The date of the most recent update to the clinical trial dataset.
     type: string
 
+  most_recent_update_datetime:
+    description: >
+      Datetime format for the most recent update to the clinical trial dataset.
+    $ref: "_definitions.yaml#/datetime"
```

Comment:
can you make it required so that it will always be there and we can rely on it for the order.